src/utilities/elsst.ts

```diff
@@ -25,17 +25,25 @@ export async function getELSSTTerm(labels: string[], lang: string): Promise<Term
   };
 
   // Request term from ELSST, the label is encoded
-  const response = await fetch(`${window.location.origin}/api/elsst/_terms`, {
-    method: "POST",
-    headers: {
-      "Content-Type": "application/json"
-    },
-    body: JSON.stringify(body)
-  });
+  try {
+    const response = await fetch(`${window.location.origin}/api/elsst/_terms`, {
+      method: "POST",
+      headers: {
+        "Content-Type": "application/json"
+      },
+      body: JSON.stringify(body),
+      signal
+    });
 
-  // Match potentially found - try to extract the URI
-  if (response.ok) {
-    return await response.json() as TermURIResult;
+    // Match potentially found - try to extract the URI
+    if (response.ok) {
+      return await response.json() as TermURIResult;
+    }
+  } catch (error) {
+    // Fetch was aborted
+    if (error instanceof DOMException && error.message === 'The user aborted a request.'){
+      return {};
+    }
   }
```

Comment:
What happens if `error` is not a `DOMException`, or the message is not as expected? There appears to be no reason for this check. Is logging missing?

Reply:
Can we easily log errors caught by this in a way that it would be useful? I'm not too familiar with Winston yet. Not that this particular error should be logged so it would make more sense to swap to <code>if (!(error instanceof DOMException && error.message === 'The user aborted a request.'))</code> and log errors that return true here.

I added this for catching errors that were caused by fetch getting aborted in <code>componentWillUnmount</code> (e.g. navigated before ELSST term fetching was finished) but yes, right now it's not really doing anything since <code>{}</code> is returned anyway for all kinds of errors or response not being ok.

Reply:
This is browser side, so using `console.log()` (and friends) should be fine

Reply:
Makes sense. I added logging for abort error also but on debug level:
```JS
    // Log error on debug level if it was because the fetch got aborted
    if (error instanceof DOMException && error.message === 'The user aborted a request.'){
      console.debug(`ELSST term fetching was aborted (${error})`);
    } else {
      console.log(`Error while fetching ELSST terms: ${error}`);
    }
```
We're not really expecting other errors so I guess it's fine to have console.log there even in production since it shouldn't produce unwarranted noise in console.